addons/src/createNumberMask.js

```diff
@@ -46,7 +52,8 @@ export default function createNumberMask({
 
     integer = (includeThousandsSeparator) ? addThousandsSeparator(integer, thousandsSeparatorSymbol) : integer
 
-    mask = convertToMask(integer)
+    mask = isNegative ? [minusRegExp] : []
```

Comment:
I don't think we need `minusRegExp`, we can use `minus` literal string here and it would be the same.